src/resources/subjects.js

```diff
@@ -0,0 +1,36 @@
+import createResourceDescription from './util/createResourceDescription'
+
+export const subjects = createResourceDescription(
+  'subjects',
+  {
+    includes: [
+      'questions',
+      'subject_category',
+      'languages',
+      'languages.translations',
+    ],
+    filters: [
+      'category_id',
+      'external_id',
+      'schedulable_at_office',
+      'schedulable_with_contact',
+      'schedulable',
+    ],
+  },
+  {
+    questions: createResourceDescription(
+      'questions',
+      {
+        includes: ['options', 'languages', 'languages.translations'],
+        filters: ['category'],
```

Comment:
remove the 'category' filter on questions
- there are only two legal values, `identification` and `subject`
- `identification` is never referenced in code, nor is there any question in any DB with a value that isn't 'subject'
==> only possible value for this field is `subject`

- we'll remove this field from the DB/php API later, but don't expose this filter now